sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/aio/_patch.py

```diff
@@ -85,13 +103,9 @@ def __init__(
         # If the credential is the typical TokenCredential, then construct the authentication policy
         # the normal way.
         else:
-            credential_scopes = kwargs.pop("credential_scopes", ["https://confidential-ledger.azure.com/.default"])
-            kwargs["authentication_policy"] = kwargs.get(
-                "authentication_policy",
-                policies.AsyncBearerTokenCredentialPolicy(credential, *credential_scopes, **kwargs),
-            )
+            kwargs["authentication_policy"] = auth_policy
 
         # Customize the underlying client to use a self-signed TLS certificate.
-        kwargs["connection_verify"] = kwargs.get("connection_verify", ledger_certificate_path)
```

Comment:
This looks like another place where we have removed kwarg override behaviour?

Reply:
Yep, just restored the original behavior